lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -532,32 +548,31 @@ func processAnonymousVolumes(volumes map[string]struct{}) ([]volumeFields, error
 
 // processSpecifiedVolumes parses fields for volumes specified explicitly during a
 // create/run (such as -v volume-name:/dir) and creates them if they don't already exist.
-func processSpecifiedVolumes(volumes []string) ([]volumeFields, error) {
-	var volumeFields []volumeFields
+func processSpecifiedVolumes(volumes []string) ([]volumeFields, error, error) {
+	var (
+		volumeFields []volumeFields
+		createErr    error
+	)
 	vol := &Volume{}
```

Comment:
This variable isn't returned and can be scoped to where it's used.  Can you move this closer to where we need it?